wagtailio/project_styleguide/templates/patterns/components/streamfields/looping_video_block/looping_video_block.html

```diff
@@ -0,0 +1,9 @@
+{% load wagtailcore_tags %}
+
+<div class="home-embed">
+    <div class="responsive-object" style="padding-bottom: 56.25%;">
+        <video {% if value.autoplay %}autoplay loop muted{% else %}controls{% endif %}>
+            <source type="{{ value.video.type }}/{{ value.video.file_extension }}" src="{{ value.video.url }}">
```

Comment:
this is not quite correct. https://github.com/torchbox/wagtailmedia/blob/main/src/wagtailmedia/models.py#L119-L120

best to iterate `value.video.sources` and compile this

Reply:
<details> <summary>Here's a screenshot</summary>
    
![image](https://user-images.githubusercontent.com/7713776/192992688-c2ad827f-2edf-407f-b792-910e45673acb.png)

</details>